Zappy_IA_Report/DTC/Video/Tutorial1.py

```
@@ -0,0 +1,19 @@
+import pandas as pd
+from skelarn.model_selection import train_test_split
+
+df = pd.read_csv("data/dataset.csv", encodings = 'latin-1')
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)